CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.1 (2020-07-29)
+
+- Added a peer dependency on `@skyux/data-manager`. [#64](https://github.com/blackbaud/skyux-ag-grid/pull/64)
```

Comment:
Should adding a peer dependency bump this up to the next minor release `4.2.0` ?

Reply:
I don't thiiiink so. The peer dep should have been added in 4.1.0 and there is an error since it wasn't. So I kind of view it as a bug fix?